package.go

```diff
@@ -11,6 +12,7 @@ import (
 
 // A Package holds the typedef, function, and enum definitions for a Go package.
 type Package struct {
+	Prefix  string
 	Name    string
```

Comment:
Sure. Either in a follow up PR, or feel free to push to this branch.

It looks like there are a couple that may be unused now.